Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM ubuntu:22.04
+
+# Install essential build tools
+RUN apt-get update && \
+    apt-get install -y \
+        build-essential \
+        libzstd-dev \
+        vim \
+        cmake \
+        gawk \
+        autoconf \
+        file \
+        unzip \
+        zip \
+        git \
+        jtreg7 \
+        openjdk-21-jdk \
+        libasound2-dev \
+        libcups2-dev \
+        libfontconfig1-dev \
+        libx11-dev libxext-dev libxrender-dev libxrandr-dev libxtst-dev libxt-dev && \
+    apt-get clean && rm -rf /var/lib/apt/lists/*
+
+# Working directory
+WORKDIR /home/jeandle/
```

Comment:
Do we need this while we have define the user's working directory in the document.